web/impact/impact/v1/serializers/office_hours_serializer.py

```diff
@@ -54,17 +56,25 @@ def validate(self, attrs):
                 'end_date_time': INVALID_SESSION_DURATION})
 
         return attrs
-        
-    def validate_mentor(self, mentor):
-        user = self.context['request'].user
-        if not is_employee(user):
-            return user
+
+    def is_allowed_mentor(self, mentor):
+        staff_user = self.context['request'].user
         roles = [UserRole.MENTOR, UserRole.AIR]
-        is_allowed_mentor = mentor.programrolegrant_set.filter(
+        if staff_user == mentor:
```

Comment:
This if statement is a bit confusing since it is checking if a staff_user is a mentor.
Could you rename it better